packages/strapi-admin/admin/src/containers/AdminPage/index.js

```diff
@@ -134,6 +142,8 @@ export class AdminPage extends React.Component { // eslint-disable-line react/pr
     }
   }
 
+  hasUserPluginLoaded = (props) => typeof get(props.plugins.toJS(), ['users-permissions', 'hasAdminUser']) !== 'undefined';
```

Comment:
```JS
hasUserPluginLoaded = (props) => !isUndefined(get(props.plugins.toJS(), ['users-permissions', 'hasAdminUser']));
```
Can't we do a smart check on the plugin itself in this case ? rather then an attribute inside of the plugin :thinking:

Reply:
I've looked at the code a bit more, it should be enough to do this check:

`hasUserPluginLoaded = (props) => typeof get(props.plugins.toJS(), ['users-permissions']) !== 'undefined';`

Since it's only added when `PLUGIN_LOADED` is dispatched.

What do you think about that?